_includes/eipfeedcontent.html

```diff
@@ -0,0 +1,22 @@
+<p><strong>EIP-#{{ eip.eip }}: {{ eip.title }}</strong> is in {{ eip.status }} status. It is authored by {{ eip.author | escape }} and was originally created {{ eip.created | date: '%Y-%m-%d' }}. It is in the {{ eip.category }} category of type {{ eip.type }}.</p>
```

Comment:
What's the motivation behind a custom template here? Why not just include the EIP itself in the feed - or perhaps only the metadata section?

Reply:
The metadata needs to get built up in this way and so needs to be "printed". I use this with some switches in the Last Call as well, and find that this human readable header is a useful way to consume in an RSS feed.

Note: I want to do the equivalent of this "all" feed as a JSON file as well, which will just be the metadata / content for consumption by machines.